config/locales/de.helpers.yml

```diff
@@ -76,19 +76,31 @@ de:
     have_fun: "Viel Erfolg beim Training!"
     closing: "%{appname} Team"
   match_mailer:
-    next_game: "dein Spiel (%{team_home} vs. %{team_away}) im Event %{name}"
+    start_time: "Wann? Am %{start_date} um %{start_time} Uhr"
+    place: "Wo? %{place}"
+    here: "Hier"
+    apply_changes_html: "%{match_link} kannst du die Änderungen vornehmen."
+    add_results_html: "%{match_link} kannst du die Ergebnisse eintragen."
     match_notification:
-      info: "startet morgen um %{start_time}. Ihr trefft euch am folgenden Ort: %{place}"
-      subject: "Dein Spiel startet morgen"
+      subject: "Dein Spiel gegen %{team_name} im Event %{event_name} startet morgen."
+      info_html: "dein Spiel gegen %{team_link} im Event %{event_link} startet morgen."
+      info_txt: "dein Spiel gegen %{team_name} im Event %{event_name} startet morgen."
+      have_fun: "Viel Spaß!"
     match_canceled:
-      info: "wurde abgesagt."
-      subject: "Dein Spiel wurde abgesagt"
+      subject: "Dein Spiel gegen %{team_name} im Event %{event_name} wurde abgesagt."
+      info_html: "dein Spiel gegen %{team_link} im Event %{event_link} wurde abgesagt."
+      info_txt: "dein Spiel gegen %{team_name} im Event %{event_name} wurde abgesagt."
+      have_fun: "Viel Erfolg!"
     match_scheduled:
-      info: "wurde geplant."
-      subject: "Dein Match wurde geplant"
+      subject: "Dein Spiel gegen %{team_name} im Event %{event_name} wurde geplant."
```

Comment:
I'm aware that the original message described in the user story #513 

> dein Spiel gegen `oponent team` in `event name` findet am `datum` `ort` statt.

was discarded in favor of of simply writing the place and location in separate lines. However, using something like “… findet bald statt” would sound more humane to my mind. 🙂